contracts/Market.sol

```diff
@@ -735,12 +752,11 @@ contract Market is
    * - Can only be used when this contract is not paused.
    * - Can only be used if this contract has been granted approval to transfer the sender's ERC20 tokens.
    * @param recipient The address to which the certificate will be issued.
-   * @param amount The total purchase amount in ERC20 tokens. This is the combined total price of the removals being
+   * @param amount The total amount of Removals to purchase. This is the combined total price of the removals being
    * purchased and the fee paid to Nori.
    */
   function swap(address recipient, uint256 amount) external whenNotPaused {
-    uint256 certificateAmount = this
-      .calculateCertificateAmountFromPurchaseTotal({purchaseTotal: amount});
+    uint256 certificateAmount = amount;
```

Comment:
same thinking on potentially not assigning this variable here and in a few other places